_integration-tests/validator/validator.go

```diff
@@ -156,17 +157,28 @@ func main() {
 		os.Exit(1)
 	}
 
-	resp, err := http.Get(*surl)
-	if err != nil {
+	var fakeTraces []map[string]any
+	if url, err := url.Parse(*surl); err == nil && url.Scheme == "file" {
+		file, err := os.Open(url.Path)
+		if err != nil {
+			fmt.Printf("Failed to open traces file %q: %v", url.Path, err)
+			os.Exit(1)
+		}
+		defer file.Close()
+		if fakeTraces, err = assembleTraces(file); err != nil {
+			fmt.Printf("Failed to decode traces from %q: %v\n", url.Path, err)
+			os.Exit(1)
+		}
+	} else if resp, err := http.Get(*surl); err != nil {
```

Comment:
Added support for `file://` URLs there in case you want to just use a file that's on the local FS.